spec/text-editor-presenter-spec.coffee

```diff
@@ -2227,6 +2099,407 @@ describe "TextEditorPresenter", ->
         expectStateUpdate presenter, -> presenter.setFocused(false)
         expect(presenter.getState().focused).toBe false
 
+    describe ".gutters", ->
+      describe ".scrollHeight", ->
```

Comment:
Seems like a lot of these properties are the same is with `.lineNumberGutter`. Makes me wonder if `.lineNumberGutter` should instead be nested underneath the `.gutters` state, like `.gutters.lineNumber`. Any thoughts on that?